src/modules/pos/validator/components/ChangeCommission/Form/ChangeCommissionForm.js

```diff
@@ -33,16 +46,45 @@ export const ChangeCommissionForm = ({ nextStep }) => {
     });
   };
 
-  const newCommissionParam = convertCommissionToNumber(newCommission);
-  const isFormValid = newCommissionParam && newCommissionParam >= 0 && newCommissionParam <= 10000;
   const formProps = {
     moduleCommand: MODULE_COMMANDS_NAME_MAP.changeCommission,
-    params: { newCommission: newCommissionParam },
-    fields: { newCommission },
-    isFormValid,
+    params: { newCommission: newCommission.numericValue },
+    fields: { newCommission: newCommission.value, token },
+    isFormValid: newCommission.value !== currentCommission && !newCommission.feedback,
   };
   const commandParams = {
-    newCommission: newCommissionParam,
+    newCommission: newCommission.numericValue,
+  };
+
+  const checkCommissionFeedback = (value) => {
+    let inputFeedback;
+    const newCommissionParam = convertCommissionToNumber(value);
+    const newCommissionValid = checkCommissionValidity(value, currentCommission);
+    const isFormValid =
+      value !== currentCommission &&
+      newCommissionParam &&
+      newCommissionParam >= 0 &&
+      newCommissionParam <= 10000 &&
+      newCommissionValid;
+    if (checkCommissionValidity(value, currentCommission)) {
+      if (isFormValid) {
+        inputFeedback = undefined;
+      }
+    } else if (newCommissionParam >= 0 && newCommissionParam <= 10000) {
+      inputFeedback = t('You cannot increase commission more than 5%');
+    } else {
+      inputFeedback = t('Commission range is invalid');
+    }
+    return { feedback: inputFeedback, numericValue: newCommissionParam };
+  };
```

Comment:
For the form validity you can simply run transaction dry run? it will tell you if its valid https://github.com/LiskHQ/lisk-sdk/blob/f4916ec8903b35648fd14f87d4fc2fff91ee46d0/framework/src/modules/pos/commands/change_commission.ts#L42-L87

also you can use the error message from the response, for example i tried to increase commission within 5 % but we missed the validation and dry returned error `"Can only increase the commission again 260000 blocks after the last commission increase."`.

So here is my suggestion.
- Validate the input field for valid range i.e 0-100 %, if the input range is valid numbers then use the input value and invoke transaction dry run, if its successful then you can continue. If its error then fetch the errorMessage value and show the error to end user.